src/core/Microsoft.PowerToys.Settings.UI.Lib/FZConfigProperties.cs

```diff
@@ -11,19 +12,28 @@ public FZConfigProperties()
         {
             this.FancyzonesShiftDrag = new BoolProperty();
```

Comment:
This is also true by default.

Reply:
resolved here: https://github.com/microsoft/PowerToys/pull/2786/commits/9c5d5706924cc3a674220819dd35657880723eb5